test/e2e/providers/aws/aws.go

```diff
@@ -133,9 +135,10 @@ func getLatestWindowsAMI(ec2Client *ec2.EC2, hasCustomVXLANPort bool) (string, e
 		windowsAMIFilterValue = "Windows_Server-2019-English-Full-ContainersLatest-????.??.??"
```

Comment:
As the date will be hard-coded, why not do it here (`????.??.??`)? And how different is this approach from hard-coding the specific AMI id?

Reply:
@jrvaldes like I explained we can't hard-code as the  AMI is different for each region. 
Also, this is a temp fix we will be getting rid of it as soon as the new AMI works or we identify the root cause.  

@sebsoto you are 100% right. Once we make CI green, next task is to see how we can escalate AMI issue n fix it